client/landing/subscriptions/components/link/link.tsx

```diff
@@ -0,0 +1,19 @@
+import { Link as RouterLink, LinkProps as RouterLinkProps } from 'react-router-dom';
+import {
+	useSubscriptionManagerContext,
+	SubscriptionsPortal,
+} from '../subscription-manager-context';
+
+type LinkProps = Omit< RouterLinkProps, 'to' > &
+	Omit< React.HTMLProps< HTMLAnchorElement >, 'ref' > & { active?: boolean };
+
+const Link: React.FunctionComponent< LinkProps > = ( { active = false, ...props } ) => {
+	const { portal } = useSubscriptionManagerContext();
+
+	if ( active || portal === SubscriptionsPortal ) {
+		return <RouterLink to={ props.href || '' } { ...props } />;
+	}
+	return <a { ...props }></a>;
```

Comment:
If I'm understanding this correctly, this means that inside Reader this just renders normal `<a>` tags? But somehow Reader catches clicks on these to turn them into SPA-style links. TIL.

Reply:
Yes, Calypso catches all `<a>` and SPA-route them.